lib/shcore-sys/src/lib.rs

```diff
@@ -1,9 +1,16 @@
 // Copyright © 2015, Peter Atashian
 // Licensed under the MIT License <LICENSE.md>
 //! FFI bindings to shcore.
-#![no_std]
-#![experimental]
+#![cfg(windows)]
 extern crate winapi;
 use winapi::*;
 extern "system" {
+    //Need to verify return value and HMONITOR handle before enabling
```

Comment:
I was stuck with HRESULT, but it turns out it was already implemented. Fixing now :)